### ISSUE TITLE
Protocop or Protokop

### ISSUE THREAD

**Issue by kennethmorton** [cbizon/protocop#39]
Currently the repo name and my (not yet merged) code is Protocop. The proposal and README is Protokop.

Let's pick one. I vote C

**Comment by cbizon** [comment]
Yeah, the proposal is ROBOKOP, and I thought it would be mildly amusing for the POC to protokop but then I accidentally changed k->c when I made the repo, so now it's protocop.  Evolution in action? 

+1 c for now, I'll change the readme file if we reach consensus.
